src/theme/DocItem/index.js

```diff
@@ -51,36 +53,40 @@ function Headings({ headings, isChild }) {
 
 function DocItem(props) {
   const { siteConfig = {} } = useDocusaurusContext();
-  const { url: siteUrl } = siteConfig;
+  const { url: siteUrl, title: siteTitle } = siteConfig;
   const { content: DocContent } = props;
   const { metadata } = DocContent;
   const {
     description,
     title,
     permalink,
-    image: metaImage,
     editUrl,
     lastUpdatedAt,
     lastUpdatedBy,
-    keywords,
     version,
     source
   } = metadata;
   const {
     frontMatter: {
+      image: metaImage,
+      keywords,
       hide_title: hideTitle,
       hide_table_of_contents: hideTableOfContents
     }
   } = DocContent;
 
   const issueTitle = `Issue with "${title}" in ${source}`;
-  const issueUrl = `https://github.com/demisto/content-docs/issues/new?labels=documentation&template=developer-documentation-issue.md&title=${issueTitle}`;
+  const issueUrl = `https://github.com/PaloAltoNetworks/cortex.pan.dev/issues/new?labels=documentation&template=developer-documentation-issue.md&title=${issueTitle}`;
```

Comment:
I think this is not correct

Reply:
Fixed this locally, haven’t pushed the new built yet. 

In terms of needing to change the templates every time we upgrade the answer is yes. As upgrades come through, it changes the components so we need to reswizzle the components and if a particular component is changed we have to incorporate the necessary changes into the swizzled component. Nothing changed on my end debug wise, will push a new update through in the morning and prolong the build.